src/components/textarea/textarea.tsx

```diff
@@ -0,0 +1,321 @@
+import {
+  Component,
+  h,
+  Prop,
+  Element,
+  Event,
+  EventEmitter,
+  VNode,
+  Watch,
+  Method,
+  Host,
+  State
+} from "@stencil/core";
+import { connectForm, disconnectForm, FormComponent, HiddenFormInputSlot } from "../../utils/form";
+import { connectLabel, disconnectLabel, getLabelText, LabelableComponent } from "../../utils/label";
+import { getSlotted } from "../../utils/dom";
```

Comment:
Can you remove use of `getSlotted()` as it is deprecated now. You can see the deprecation notes on how to migrate.